docs/user-guide/update-demo/index.md

```diff
@@ -13,11 +13,11 @@ here](https://github.com/kubernetes/kubernetes.github.io/tree/{{page.docsbranch}
 
 ### Step Zero: Prerequisites
 
-This example assumes that you have forked the repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
+This example assumes that you have forked the docs repository and [turned up a Kubernetes cluster](/docs/getting-started-guides/):
 
 ```shell
-$ cd kubernetes
-$ ./cluster/kube-up.sh
+$ git clone -b {{page.docsbranch}} https://github.com/kubernetes/kubernetes.github.io

```

Comment:
@devin-donnelly the [kubernetes repo](https://github.com/kubernetes/kubernetes/tree/master/docs/user-guide/update-demo) no longer contains the user-guide docs. As a result cd kubernetes is no longer accurate as the rest of the update demo documentation assumes the user docs are under the docs directory. So this patch(include) is fixing the issue that that current assumptions are incorrect.

@devin-donnelly Now what do you specifically want taken out? Only the `-b {{page.docsbranch}}` part? Or the whole git clone statement?

Please be as specific as possible. I usually do not write docs and just happened to stumble upon an issue with the docs which I fixed in this patch..